phpunit.xml.dist

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/5.7/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.2/phpunit.xsd"
```

Comment:
This had to be updated because the 5.7 xml schema doesn't have executionOrder, and infection validates the config against the schema.

Reply:
it's better to use `xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"`

Reply:
```suggestion
         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.2/phpunit.xsd"
```